server/src/test/java/org/elasticsearch/cluster/metadata/DataStreamLifecycleTests.java

```diff
@@ -117,6 +117,7 @@ public void testXContentSerializationWithRolloverAndEffectiveRetention() throws
             RolloverConfiguration rolloverConfiguration = RolloverConfigurationTests.randomRolloverConditions();
             DataStreamGlobalRetention globalRetention = DataStreamGlobalRetentionTests.randomGlobalRetention();
             ToXContent.Params withEffectiveRetention = new ToXContent.MapParams(DataStreamLifecycle.INCLUDE_EFFECTIVE_RETENTION_PARAMS);
+            boolean systemDataStream = randomBoolean();
```

Comment:
Looks like this line is a left-over change that isn't necessary anymore.

Reply:
Yep, gone now.